crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -269,18 +274,39 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for CreateTransactionRequest {
     }
 }
 
-impl TryFrom<&types::PaymentsCancelRouterData> for CancelTransactionRequest {
+impl TryFrom<&types::PaymentsCancelRouterData> for CancelOrCaptureTransactionRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(item: &types::PaymentsCancelRouterData) -> Result<Self, Self::Error> {
-        let transaction_request = TransactionVoidRequest {
+        let transaction_request = TransactionVoidOrCaptureRequest {
+            amount: item.request.amount,
             transaction_type: TransactionType::Void,
             ref_trans_id: item.request.connector_transaction_id.to_string(),
         };
 
         let merchant_authentication = MerchantAuthentication::try_from(&item.connector_auth_type)?;
 
         Ok(Self {
-            create_transaction_request: AuthorizedotnetPaymentCancelRequest {
+            create_transaction_request: AuthorizedotnetPaymentCancelOrCaptureRequest {
+                merchant_authentication,
+                transaction_request,
+            },
+        })
+    }
+}
+
+impl TryFrom<&types::PaymentsCaptureRouterData> for CancelOrCaptureTransactionRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsCaptureRouterData) -> Result<Self, Self::Error> {
+        let transaction_request = TransactionVoidOrCaptureRequest {
+            amount: Some(item.request.amount_to_capture),
+            transaction_type: TransactionType::Capture,
```

Comment:
request type is CancelOrCaptureTransactionRequest but transaction_type is always `Capture`?

Reply:
the structure for capture and cancel are same . This is capture flow that's why transaction type is Capture